noxfile.py

```diff
@@ -37,7 +37,6 @@ def set_environment_variables(env_dict, session):
 def run_coverage(session):
     """Run the coverage tests and generate an XML report."""
     set_environment_variables(PYBAMM_ENV, session=session)
-    session.install("setuptools", silent=False)
```

Comment:
Some of these were added for a bug with pybtex, so if you start seeing errors, then these lines might be needed again

Reply:
Looks like they are passing, so these are fine